src/Lint.jl

```diff
@@ -335,7 +341,7 @@ function lintinclude(ctx::LintContext, file::AbstractString)
 end
 
 """
-Lint all *.jl files at a given directory.
+Lint all \*.jl files at a given directory.
```

Comment:
Sorry I forgot it here. This is atom-language-julia bug of syntax highlighting. I just use the backslash to show the rest of the file highlighting properly.